modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/frontend/taglib/servlet/taglib/AccountUserAccountEntriesScreenNavigationEntry.java

```diff
@@ -103,4 +103,7 @@ public void render(
 	@Reference
 	protected JSPRenderer jspRenderer;
 
+	@Reference
+	protected UserPermission userPermission;
```

Comment:
@julschong Why this is protected? No one extends this class.

Reply:
@tinatian Hi Tina, I think I just followed the JSPRenderer service reference already there. I'll make this private and resubmit it